bin/katello-certs-check

```diff
@@ -114,6 +114,17 @@ function check-cert-ca-flag () {
     fi
 }
 
+function check-passphrase () {
+    printf "Checking for private key passphrase: "
+    CHECK=$(cat $KEY_FILE | grep ENCRYPTED)
+    if [[ $? == "0" ]]; then
+        error 2 "The $KEY_FILE contains a passphrase, please recreate the csr and key without a passphrase."
```

Comment:
Yeah, I will do that, I see you can do that in the manpage here:

https://www.openssl.org/docs/man1.1.1/man1/rsa.html

I should have looked at that first. I will do that and change it to a warning letting the user know since we want to be transparent.

Reply:
Yes, instructing the user is probably a better approach because it's safer.